tensor_comprehensions/pybinds/pybind_options.cc

```diff
@@ -17,6 +17,8 @@
 #include <string>
 #include <vector>
 
+#include <google/protobuf/text_format.h>
```

Comment:
Changes to this file should land in a first commit with a proper commit message.

For the python part, depending on whether or not the test can run in OSS, it should either be completely omitted if it doesn't run (i.e. it is dead code) or it should be also added to the CI in the same PR.

Commit message is completely off.